src/transformers/models/vit/image_processing_vit_fast.py

```diff
@@ -246,6 +246,7 @@ def preprocess(
         image_mean = image_mean if image_mean is not None else self.image_mean
         image_std = image_std if image_std is not None else self.image_std
         size = size if size is not None else self.size
+        return_tensors = "pt" if return_tensors is None else return_tensors
```

Comment:
Is this kind of breaking change?

(I am also curious what if a user is using TF/Flax model while their environment has torch/torchvision installed. Is the fast image processor will be used by default and will return torch tensor by default ..?)

Reply:
Oh I see, yes this might be a problem. In general, I'm not too sure why it was decided to constrain fast image processors to output only torch tensors. Would be glad to know if there was a reason for that, otherwise it might be something we would want to reconsider.

Reply:
TBH I am not even sure TF is used for image models! Fine by me!